src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResourceIntTest.kt

```diff
@@ -2987,7 +2988,238 @@ class SentencingResourceIntTest : IntegrationTestBase() {
     }
 
     @AfterEach
-    internal fun deletePrisoner() {
+    internal fun deleteSentence() {
+      repository.deleteSentenceByBookingId(latestBookingId)
```

Comment:
as a matter of interest if you just delete the offender won't it cascade down to case and charge?

Reply:
no it doesn't - I could look into why